hbase-protocol-shaded/src/main/protobuf/server/region/Admin.proto

```diff
@@ -219,6 +219,12 @@ message RollWALWriterResponse {
   repeated bytes region_to_flush = 1;
 }
 
+message ArchiveWALRequest {
```

Comment:
Should this be renamed to go with the updated method name ?

Reply:
Will fix later. Thanks.

Reply:
Thought again, it should be a common name if we want to improve it in future, such as archive a specify log file, archive all log files, etc.

Reply:
If it make sense, i will remove the "earliest" from method name also.

Reply:
I think ArchiveWALResponse should contain the status of archival (including error).

Reply:
> I think ArchiveWALResponse should contain the status of archival (including error).

Make sense.

Reply:
Checked the existing api in Admin.java, such as flushRegion, compactRegion, seems currently we dont use status, if there is something wrong at server side, we notice client by throw ServiceException. 
Though the FlushRegionResponse has a flushed flag, but does not be used at client.
Maybe we should keep consist?